test/verify/check-machines

```diff
@@ -3622,17 +3623,17 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
             b.wait_not_present("#vm-subVmTest1-snapshots")
             return
 
-        b.wait_present("#vm-subVmTest1-snapshot-0-current")
-        b.wait_not_present("#vm-subVmTest1-snapshot-1-current")
+        b.wait_present("#vm-subVmTest1-snapshot-1-current")
```

Comment:
I don't understand this change :( 

snapshot-0 should be the current, at least that is what we assert two lines bellow

Reply:
Ah yes, that's confusing. in this css selector, snapshot 1 means the id in a list of snapshot, os a snapshot with and index 1. In this case, the list is sorted as [snapshot1, snapshot0], so snapshot0 has an id 1 and therefore has a #vm-subVmTest1-snapshot-1-current.

That's hella confusing, I know. But in this case I cannot use snapshot name as an identifier, since it's an optional parameter for snapshots.

Perhaps I could use different names for snapshots in these tests, like snapshotA and snapshotB

Reply:
Oh right, that was utterly confusing!